.wordlist.txt

```diff
@@ -279,6 +279,7 @@ multithreading
 Mumble
 Munin
 Mycroft
+myDietpiSystem
```

Comment:
Please wrap such kind of words into code tags `myDietpiSystem`, same below with `USESTATIC` (respectively the whole settings) so that they are excluded from spellcheck in the first place and do not need to be in the wordlist.

Reply:
Done.